### ISSUE TITLE
Feature request: Multi-key keyboard shortcut (chord) map

### ISSUE THREAD

**Issue by Bill-Stewart** [notepad-plus-plus/notepad-plus-plus#5191]
Feature request:

Many other editors (e.g., EMACS, Visual Studio, VS Code) allow mappings of multi-key keyboard shortcuts, sometimes called "chords." I don't mean like <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>X</kbd>, I mean like <kbd>Ctrl</kbd>+<kbd>K</kbd> followed by <kbd>Ctrl</kbd>+<kbd>Q</kbd>.

Example in Visual Studio Code JSON keyboard configuration:

    {
      "key": "ctrl+k q",
      "command": "workbench.action.closeActiveEditor",
      "when": "editorTextFocus"
    },
    {
      "key": "ctrl+k ctrl+q",
      "command": "workbench.action.closeActiveEditor",
      "when": "editorTextFocus"
    }

In other words: When I press <kbd>Ctrl</kbd>+<kbd>K</kbd> followed by <kbd>Ctrl</kbd>+<kbd>Q</kbd> (or <kbd>Ctrl</kbd>+<kbd>K</kbd> followed by <kbd>q</kbd> or <kbd>Q</kbd>), VS Code closes the current file.

Supporting mappings like these would be very useful for users of other editors that support these kinds of multi-key (chord) mappings.

**Comment by cfarrar** [comment]
I would vote for this as well.

**Comment by LuxiFerrit** [comment]
I am using EMACS for decades ... this would help a lot to switch to a newer editor! Begging for this with all my heart! ;-)
